packages/calcite-components/src/components/time-picker/time-picker.tsx

```diff
@@ -221,6 +223,8 @@ export class TimePicker
 
   @Listen("blur")
   hostBlurHandler(): void {
```

Comment:
Would you be able to remove `host` here too since you're already updating the function?